src/components/DataCollector/index.js

```diff
@@ -51,7 +52,8 @@ createDataCollector.namespace = "DataCollector";
 
 createDataCollector.configValidators = {
   collectionUrl: {
-    defaultValue: "https://edgegateway.azurewebsites.net"
+    defaultValue: "https://edgegateway.azurewebsites.net",
+    validate: secureUrl
```

Comment:
Should `collectionUrl` be `collectionDomain` instead (I think so)? And if that's the case, should we validate that it's a domain and not any beyond that?

Reply:
Good point. The only thing we get out of enforcing that https is specified is that it is clear that the end-point needs to be SSL. I don't know if that is enough to having to specify the protocol. If not this would instead be highlighted by requests failing.